### ISSUE TITLE
chore: improve logging when fails to connect to a node

### ISSUE THREAD

**Issue by weboko** [waku-org/js-waku#1408]
This is a **bug report/feature or change request/support request**

## Problem

Sometimes when a user tries to establish connection to a default node 
`await createLightNode({ defaultBootstrap: true })` and it fails it prints vague log in the console:
```
examples.waku.org/:1 Uncaught (in promise) Error dialing peer 16Uiu2HAmVkKntsECaYfefR1V2yCR79CegLATuTPE6B9TxgxBiiiA : Error deleting undialable peer 16Uiu2HAmVkKntsECaYfefR1V2yCR79CegLATuTPE6B9TxgxBiiiA from peer store - TypeError: Cannot read properties of undefined (reading '0')
```

## Proposed Solutions

Improve logging: show what operation on the side of `js-waku` failed and why. 

## Notes

Probably a follow up to https://github.com/waku-org/js-waku/pull/1380

**Comment by fryorcraken** [comment]
May need clarification on how to reproduce.

**Comment by weboko** [comment]
Still seems to be happening. I can easily repro it by opening https://examples.waku.org/light-chat/ and waiting ~2 minutes.

**Comment by fryorcraken** [comment]
> unable to repro :(
> 
> ![Image](https://user-images.githubusercontent.com/35004822/256499433-ba172388-eae4-4e21-a556-332da7a5d253.png)

Yes because a lot of improvements have been delivered on nwaku to avoid connectiong hogging AND ensure that websocket remains stable.

I would recommend to put an unreachable address on purpose to reproduce the error and then improve the logging.

**Comment by weboko** [comment]
Tried again - doesn't seem to be happening.
